atd-etl/cris_import/lib/mappings.py

```diff
@@ -77,7 +77,7 @@ def no_override_columns():
             "prsn_nbr",  # key column
             "prsn_type_id",  # key column (but why?)
             "prsn_occpnt_pos_id",  # key column (but why?)
-            "injury_severity",
+            "prsn_injry_sev_id",
```

Comment:
this was an issue we identified which is unrelated to this PR. i'm going to start spin out a separate task of investing whether or not we've overwritten the `prsn_injry_sev_id` in cases where we had not intended to 😵